xml/Microsoft.VisualBasic.Compatibility.VB6/BaseControlArray.xml

```diff
@@ -680,7 +680,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Design environments typically use this method to start the initialization of a component that is used on a form or used by another component. The <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit> method ends the initialization. Using the `BeginInit` and <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit> methods prevents the control from being used before it is fully initialized. The initialization occurs at run time.  
+ Design environments typically use this method to start the initialization of a component that is used on a form or used by another component. The <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit%2A> method ends the initialization. Using the `BeginInit` and <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit%2A> methods prevents the control from being used before it is fully initialized. The initialization occurs at run time.  
```

Comment:
Both links are still broken, and I'm not sure why. I also tried changing them in a PR a few months ago, and could neither the original syntax that you changed nor the addition of %2A seemed to work.

Reply:
😢 

ok, I'll create a bug for this then